### DISCUSSION TITLE
Texture render looks pale compared to EEVEE or Workbench

### DISCUSSION THREAD

**Discussion by nz21f** [bnpr/Malt#226] [Q&A]
i downloaded latest BlenderMalt version
I like lineart system and cell shading on Blendermalt but with texture seems too pale. i also want double face feature. mesh backface not visible on my test model
![image](https://user-images.githubusercontent.com/70623710/151668693-c2193345-13bb-46c4-b66f-9710080769f1.png)

diffuse gradient color looks fine
![image](https://user-images.githubusercontent.com/70623710/151668581-bc0937eb-fa8f-477d-becb-21067fbfda0a.png)

but with texture looks too pale compared to EEVEE/workbench render
![image](https://user-images.githubusercontent.com/70623710/151668533-3193da91-6c42-4472-8b94-e94d276ed167.png)
![image](https://user-images.githubusercontent.com/70623710/151668524-e6133a12-8c9d-44b2-9bbf-897fe7c1136f.png)
![image](https://user-images.githubusercontent.com/70623710/151668529-554d7472-dc86-4faa-a5b8-56f816a48bde.png)

**Comment by pragma37** [comment]
Hi! You can disable backface culling in *Properties Panel > Mesh > Malt Settings > double sided*:
![imagen](https://user-images.githubusercontent.com/36610999/151671414-34ff158c-2efc-4bc5-b6f9-ff494611fb96.png)

Seems like I broke texture loading in fff960b (#215 for reference) when Float Buffer is not enabled.
82bfc770651cdf5523b797d9010ac172c60a886c should fix it.

If you download the latest version it should work in both cases.

**Reply by nz21f** [reply]
![image](https://user-images.githubusercontent.com/70623710/151683663-05bc9f6a-23ce-4b45-b54a-1c1ce20f12fc.png)
thank you it worked now.

is there tips for rimlight and halftone shader?

**Reply by pragma37** [reply]
There's a rim light function/node so you can just use that.
For halftone shaders, there's nothing built-in, but you can roll your own in the same way you would with EEVEE, for example.

**Comment by HSDSZ** [comment]
What? the pale image color is a bug? I thought it was the feature of MALT. I kind of getting used to the previous color style, it is more bright.
How should I modify my sampler2D image to achieve the previous pale style?

**Reply by pragma37** [reply]
Ooops! Sorry for the trouble.
If you set your textures *Color Space* to *Linear* instead of *sRGB* everything should look as it did before the fix.
